src/Grid/Column/Column.php

```diff
@@ -240,6 +240,10 @@ public function manipulateRenderCell($callback)
         return $this;
     }
 
+    public function getCallback(): mixed {
```

Comment:
Should this actually be mixed? I think we should be able to make it something like `?Callable` or maybe it also supports strings? Depends how it's used.

Reply:
Updated to be `?\Closure`